service/src/test/org/apache/hive/service/auth/TestLdapAuthenticationProviderImpl.java

```diff
@@ -250,7 +255,11 @@ public void testAuthenticateWhenCustomQueryFilterFailsAndUserFilterPasses() thro
         "(&(objectClass=person)(|(memberOf=CN=Domain Admins,CN=Users,DC=apache,DC=org)(memberOf=CN=Administrators,CN=Builtin,DC=apache,DC=org)))");
     conf.setVar(HiveConf.ConfVars.HIVE_SERVER2_PLAIN_LDAP_USERFILTER, "user3");
 
-    when(search.findUserDn("user3")).thenReturn("cn=user3,ou=PowerUsers,dc=mycorp,dc=com");
+//    Unnecessary stubbings detected in test class: TestLdapAuthenticationProviderImpl
+//    Clean & maintainable test code requires zero unnecessary code.
+//    Following stubbings are unnecessary
+
```

Comment:
Please fix this comment

Reply:
I don't know how I missed that. I fixed one of them and just left the rest. Sorry, I fixed it.